xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -561,20 +570,64 @@ void CGUIDialogPVRTimerSettings::Save()
   const CDateTime now(CDateTime::GetCurrentDateTime());
   if (!bStartSet || !bEndSet)
   {
-    if (m_startLocalTime < now)
-      m_startLocalTime = now;
-
-    if (m_endLocalTime < m_startLocalTime)
+    if (m_timerType->SupportsStartEndTime() && //i.e. has start/end time field
+        m_timerType->IsRepeating())            //but has no start day / end day spinners
```

Comment:
That the manipulation of startTime and endTime only takes place when there is a start/end clock field but no associated start/end date spinner for the user to interact with.
This is the condition that needs to be confirmed cworking using a manual repeating timer.